workflows/metatdenovo.nf

```diff
@@ -39,15 +39,18 @@ ch_multiqc_custom_config = params.multiqc_config ? Channel.fromPath(params.multi
     IMPORT LOCAL MODULES/SUBWORKFLOWS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
+
 //
 // MODULE: local
 //
 include { MEGAHIT_INTERLEAVED              } from '../modules/local/megahit/interleaved.nf'
 include { UNPIGZ as UNPIGZ_MEGAHIT_CONTIGS } from '../modules/local/unpigz.nf'
+include { UNPIGZ as UNPIGZ_FASTA_PROTEIN   } from '../modules/local/unpigz.nf'
```

Comment:
Would it perhaps be better to make sure the modules (e.g. eggnog-mapper) unzips their inputs? I think that's more inline with nf-core recommendations.

Reply:
do you mean by having a subworkflow? or having the unzip in eggnog-mapper module?

Reply:
What I was thinking of here was to include the unzipping in the module, so you don't have to unzip beforehand. I'm not really sure what nf-core recommends.

Reply:
Leave it as it is for now, and we can discuss later.